src/components/StakingVaults/Cells.tsx

```diff
@@ -32,7 +32,7 @@ export const AssetCell = ({ assetId, subText, showTeaser, postFix, onClick }: As
   const debouncedHandleMouseEnter = debounce(() => setShowPopover(true), 100)
   const handleOnMouseLeave = debouncedHandleMouseEnter.cancel
   const asset = useAppSelector(state => selectAssetByCAIP19(state, assetId))
-  const rowTitle = postFix ? `${asset.name} ${postFix}` : asset.name
+  const rowTitle = postFix ? `${asset?.name} ${postFix}` : asset?.name
```

Comment:
Maybe it's stupid, but does this mean that the rowTitle can be `undefined` because of the optional chaining? Maybe it's an edge case that probably never happen but I feel uncomfortable with this

Reply:
I think that was an edge case and something to do with the weird COSMO feature flag, i'll reset it back to how it was.